src/main/kotlin/org/malachite/estella/aplication/domain/ApplicationPayloads.kt

```diff
@@ -161,7 +161,7 @@ fun TaskResult.toTaskResultWithTestDTO(): TaskResultWithTestDTO =
         code.toBase64String(),
         this.results?.toBase64String()?: throw TaskResultNotExistException(),
         task.tests.toBase64String(),
-        task.description.toBase64String()
+        task.description
```

Comment:
czy sprawdzałeś, czy na froncie dalej działa? Bo na przykład jeśli to jest opis taska, to tam już dekodujemy dwa razy base64 z tego właśnie powodu. Jeśli zmieniasz tutaj, powinieneś też zaktualizować front